forch/port_state_manager.py

```diff
@@ -22,30 +23,38 @@ def register(func):
 class PortStateMachine:
     """State machine class that manages testing states of an access port"""
 
+    UNAUTHENTICATED = 'unauthenticated'
     AUTHENTICATED = 'authenticated'
     SEQUESTERED = 'sequestered'
     OPERATIONAL = 'operational'
     INFRACTED = 'infracted'
 
     TRANSITIONS = {
-        AUTHENTICATED: {
+        UNAUTHENTICATED: {
```

Comment:
This doesn't seem right. We should only be getting to this point after authentication, no?

Reply:
So "authenticated" is more like a transient state. Conceptually device is authenticated, but then depending the static port behavior, the port will be set to either sequestered or operational once authenticated. So the port will never be in a state  called "authenticated"...